ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -692,6 +692,21 @@ func (r *RayClusterReconciler) reconcilePods(ctx context.Context, instance *rayv
 		deleted := struct{}{}
 		numDeletedUnhealthyWorkerPods := 0
 		for _, workerPod := range workerPods.Items {
+			logger.Info("reconcilePods", "Worker Pod Name: ", workerPod.Name, " Status: ", workerPod.Status.Phase)
+			if workerPod.Status.Phase != "Running" {
+				if len(workerPod.Status.Conditions) > 0 {
+					for _, condition := range workerPod.Status.Conditions {
+						if condition.Message != "" {
+							logger.Info("reconcilePods",
+								"Worker Pod Name", workerPod.Name,
+								"Status", workerPod.Status.Phase,
+								"Condition Type", condition.Type,
+								"Condition Status", condition.Status,
+								"Message", condition.Message)
+						}
+					}
+				}
+			}
 			shouldDelete, reason := shouldDeletePod(workerPod, rayv1.WorkerNode)
 			logger.Info("reconcilePods", "worker Pod", workerPod.Name, "shouldDelete", shouldDelete, "reason", reason)
```

Comment:
Is this log not enough?

Reply:
Do you mean without line 695?

Reply:
L696

Reply:
okay then maybe I can replace L695 with the else condition and leave just the if block statement